tests/test_tui_publish_workflow_security.py

```
@@ -1348,6 +1361,169 @@ def test_stable_registry_publishers_are_exact_tag_and_artifact_bound() -> None:
+            protected_branches=False,
+            custom_branch_policies=True,
+        )
+    )
+    assert result.returncode != 0
+    assert "protected branches" in result.stderr
+
+
+def test_environment_policy_reads_non_ascii_json_with_utf8() -> None:
+    script = ROOT / "cmux-tui" / "bindings" / "verify_release_environment_policy.py"
+    spec = importlib.util.spec_from_file_location("release_environment_policy", script)
+    assert spec is not None and spec.loader is not None
```

Ruff (0.16.1)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)